neurodamus/io/synapse_reader.py

```diff
@@ -232,7 +232,8 @@ class SonataReader(SynapseReader):
     }
 
     def _open_file(self, src, population, _):
-        storage = libsonata.EdgeStorage(src)
+        hdf5_reader = libsonata.make_collective_reader(False, True)
```

Comment:
`libsonata.make_collective_reader(comm, collective_metadata, collective_transfer)`, maybe `comm` is missing ?

Reply:
Yes, it used to have a default for `MPI.COMM_WORLD` where `MPI` is `from mpi4py import MPI`.

Reply:
Thanks, changed.